kustomize/overlays/prod/manager/patches/deploy-config.yaml

```diff
@@ -8,6 +8,5 @@ data:
   EVENT_BRIDGE_DB_HOST: "event-bridge-db"
   EVENT_BRIDGE_DB_PORT: "5432"
   EVENT_BRIDGE_DB_SCHEMA: "event-bridge"
-  EVENT_BRIDGE_MANAGER_URL: "http://event-bridge:8080"
```

Comment:
Is this change related with the PR or it is just a minor cleanup?

Reply:
I think it was simply a copy/paste mistake, this config property is only for the shard and not for the manager. So I'm just cleaning this up